news.xml

```diff
@@ -444,15 +480,15 @@
     </title>
     <description>
       A <a href="http://heartbleed.com/">serious security
-      vulnerability</a> has been discovered in OpenSSL. All stable
+        vulnerability</a> has been discovered in OpenSSL. All stable
       NixOS releases prior to version
       <strong>13.10.35708.15a465c</strong> are vulnerable. (You can
       see your current version by running <tt>nixos-version</tt>.) To
       upgrade to the latest NixOS version, run <tt>nixos-rebuild
-      switch --upgrade</tt>. You can verify whether you are safe by
+        switch --upgrade</tt>. You can verify whether you are safe by
       running
 
-<pre class="code">
```

Comment:
shouldn't the start pre have the same identation as end pre? the block between could be idented

Reply:
i think my editor did this. need to somehow revert this.